jest.config.js

```diff
@@ -20,6 +20,8 @@ module.exports = {
   modulePathIgnorePatterns: [
     'node_modules'
   ],
+  testPathIgnorePatterns: [
```

Comment:
The empty array could be removed to keep the config a bit cleaner.

Reply:
Or it could be left in to make it easier to add `testPathIgnorePatterns` in future! Happy to remove it.